Update README.md

asilvino opened the pull request:
Just an update about how action hero is adding a plugin and,  a tip about how access the Swagger UI html inside the project. 
I think that These information will be useful for beginners users.

asilvino committed:
Update README.md

Just an update on how adding an plugin and a tip about how access the Swagger UI inside the project. I think that These information will be useful for first users.

evantahler commented:
Nice!

I'm so happy the plugin system ended up being more-or-less backwards compatible

asilvino committed:
actionPath not vaid fir Swagger 2.0

actionPath property is not valid for Swagger 2.0 documentation 
https://github.com/OAI/OpenAPI-Specification/blob/master/schemas/v2.0/schema.json
